src/css/_utilities.scss

```diff
@@ -115,3 +115,10 @@
   @extend %fill-parent;
   text-align: center;
 }
+
+// don't wrap whitespace for the time control and make the text container 0 width
+// fixes a bug with 'width: auto' in IE 8/9
+.vjs-ie-auto-width-fix {
```

Comment:
Thinking about this a bit more. Can we re-use the vjs-no-flex class that gets added to the player div? It'll definitely get added in IE8 and 9.